test/functional/feature_dbcrash.py

```diff
@@ -211,7 +211,9 @@ def run_test(self):
         self.crashed_on_restart = 0      # Track count of crashes during recovery
 
         # Start by creating a lot of utxos on node3
-        utxo_list = self.wallet.send_self_transfer_multi(from_node=self.nodes[3], num_outputs=5000)['new_utxos']
+        utxo_list = []
+        for _ in range(5):
+            utxo_list.extend(self.wallet.send_self_transfer_multi(from_node=self.nodes[3], num_outputs=1000)['new_utxos'])
```

Comment:
why is this needed?

Reply:
To not exceed MAX_STANDARD_TX_WEIGHT, see PR description.